config/common-packages.yml

```diff
@@ -22,6 +23,7 @@ install:
   - chkrootkit
   - koji
   - codium
+  - bubblewrap
```

Comment:
should this be bubblewrap-suid? the name will clash with the existing bubblewrap package

Reply:
I thought replacing the package is the cleanest way.

While testing locally i used `override replace`  to replace the existing package. 

I tested this on a locally built image from `recipe-silverblue-main.yml` and it worked

Reply:
I just figured that that Ublue treats these installs as overrides as well...

Reply:
blue-main.yml` and it worked
> 
> But now if i think about it i am uncertain. But how would the dependencies be solved if we change the name to `bubblewrap-suid`?

That's a fair point. This should be fine as-is and we can revisit if we run into problems later